data/00700_workload.js

```diff
@@ -1,27 +1,19 @@
-        { workload_owner_id: randomWorkloadOwnerId.id, total_cases: 15, monthly_sdrs: 10, sdr_due_next_30_days: 10, sdr_conversions_last_30_days: 10, total_community_cases: 5, total_custody_cases: 10, total_license_cases: 3, paroms_completed_last_30_days: 5, paroms_due_next_30_days: 5, license_last_16_weeks: 9, community_last_16_weeks: 10 },
-        { workload_owner_id: randomWorkloadOwnerId.id, total_cases: 15, monthly_sdrs: 10, sdr_due_next_30_days: 10, sdr_conversions_last_30_days: 10, total_community_cases: 5, total_custody_cases: 10, total_license_cases: 3, paroms_completed_last_30_days: 5, paroms_due_next_30_days: 5, license_last_16_weeks: 9, community_last_16_weeks: 10 },
-        { workload_owner_id: randomWorkloadOwnerId.id, total_cases: 15, monthly_sdrs: 10, sdr_due_next_30_days: 10, sdr_conversions_last_30_days: 10, total_community_cases: 5, total_custody_cases: 10, total_license_cases: 3, paroms_completed_last_30_days: 5, paroms_due_next_30_days: 5, license_last_16_weeks: 9, community_last_16_weeks: 10 },
-        { workload_owner_id: randomWorkloadOwnerId.id, total_cases: 15, monthly_sdrs: 10, sdr_due_next_30_days: 10, sdr_conversions_last_30_days: 10, total_community_cases: 5, total_custody_cases: 10, total_license_cases: 3, paroms_completed_last_30_days: 5, paroms_due_next_30_days: 5, license_last_16_weeks: 9, community_last_16_weeks: 10 },
-        { workload_owner_id: randomWorkloadOwnerId.id, total_cases: 15, monthly_sdrs: 10, sdr_due_next_30_days: 10, sdr_conversions_last_30_days: 10, total_community_cases: 5, total_custody_cases: 10, total_license_cases: 3, paroms_completed_last_30_days: 5, paroms_due_next_30_days: 5, license_last_16_weeks: 9, community_last_16_weeks: 10 },
-        { workload_owner_id: randomWorkloadOwnerId.id, total_cases: 15, monthly_sdrs: 10, sdr_due_next_30_days: 10, sdr_conversions_last_30_days: 10, total_community_cases: 5, total_custody_cases: 10, total_license_cases: 3, paroms_completed_last_30_days: 5, paroms_due_next_30_days: 5, license_last_16_weeks: 9, community_last_16_weeks: 10 },
-        { workload_owner_id: randomWorkloadOwnerId.id, total_cases: 15, monthly_sdrs: 10, sdr_due_next_30_days: 10, sdr_conversions_last_30_days: 10, total_community_cases: 5, total_custody_cases: 10, total_license_cases: 3, paroms_completed_last_30_days: 5, paroms_due_next_30_days: 5, license_last_16_weeks: 9, community_last_16_weeks: 10 }
-      ])
+    .then(function (workloadOwners) {
+      var workloadsToInsert = []
+      for (workloadOwner in workloadOwners) {
+        workloadsToInsert.push(Object.assign({}, workloadRow, {workload_owner_id: workloadOwners[workloadOwner].id})),
+        workloadsToInsert.push(Object.assign({}, workloadRow, {workload_owner_id: workloadOwners[workloadOwner].id}))          
+      }
+      return knex('workload').insert(workloadsToInsert)
```

Comment:
👌 Much cleaner. Thanks! We're going to need a few more in order to render capacity / reduction graph correctly though.